File: core/src/si/feri/slidegame/utils/Geolocation.java
package si.feri.slidegame.utils;

public class Geolocation {
    public double lat;
    public double lng;


    public Geolocation() {
        this(46.557314, 15.637771);
    }
    public Geolocation(double lat, double lng) {
        this.lat = lat;
        this.lng = lng;
    }

    @Override
    public String toString() {
        return "Lat: " + lat + "\nLng: " + lng;
    }
}

File: core/src/si/feri/rrizemljevid/utils/Geolocation.kt
package si.feri.rrizemljevid.utils

class Geolocation @JvmOverloads constructor(var lat: Double = 46.557314, var lng: Double = 15.637771) {
    override fun toString(): String {
        return "Lat: $lat\nLng: $lng"
    }
}